### ISSUE TITLE
[Issue] `nil` return for some elements

### ISSUE THREAD

**Issue by pidgeon777** [polirritmico/telescope-lazy-plugins.nvim#13]
I've encountered a potential issue with `telescope-lazy-plugins` that seems to return `nil` for some elements. Below are two images illustrating the problem:

![First image of the problem](https://github.com/user-attachments/assets/b80673e8-f6eb-45f6-985d-bcb1ae3fe5ce)

![Second image of the problem](https://github.com/user-attachments/assets/0afc783f-88b4-4936-b6e9-d8ddcb7afc95)

This issue seems to be associated with the following plugins:

- **LazyVim**
- **lazy.nvim**

Again, I'm not sure this is an issue or a design choice.

**Comment by polirritmico** [comment]
Hi, thanks again for the bug report.

Could you share the output of `:checkhealth telescope`?

**Comment by pidgeon777** [comment]
Here it is:

```

==============================================================================
telescope: health#telescope#check

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 14.1.0 (rev e50df40a19)
- OK fd: found fd 10.1.0

===== Installed extensions ===== ~

Telescope Extension: `aerial` ~
- No healthcheck provided

Telescope Extension: `frecency` ~
- OK nvim-web-devicons installed.
- OK ripgrep installed.

Telescope Extension: `fzf` ~
- OK lib working as expected
- OK file_sorter correctly configured
- OK generic_sorter correctly configured

Telescope Extension: `notify` ~
- No healthcheck provided

Telescope Extension: `projects` ~
- No healthcheck provided
```

Thanks again.

**Comment by polirritmico** [comment]
Please load the extension before calling the checkhealth:

 `:Telescope lazy_plugins` and then `:checkhealth telescope`.

**Comment by pidgeon777** [comment]
```
==============================================================================
telescope: health#telescope#check

Checking for required plugins ~
- OK plenary installed.
- OK nvim-treesitter installed.

Checking external dependencies ~
- OK rg: found ripgrep 14.1.0 (rev e50df40a19)
- OK fd: found fd 10.1.0

===== Installed extensions ===== ~

Telescope Extension: `aerial` ~
- No healthcheck provided

Telescope Extension: `fzf` ~
- OK lib working as expected
- OK file_sorter correctly configured
- OK generic_sorter correctly configured

Telescope Extension: `lazy_plugins` ~
- OK Telescope Lazy Plugins configuration found.
- OK Telescope Lazy Plugins configuration options found.
- OK lazy_config found: `C:\Users\<USERNAME>\AppData\Local\nvim\lua\config\lazy.lua`
- OK No problems importing plugins config specs.
- OK Found all imported plugin configurations in the module files.

Telescope Extension: `notify` ~
- No healthcheck provided

Telescope Extension: `projects` ~
- No healthcheck provided
```

**Comment by polirritmico** [comment]
Fixed by ad0cde8.

**Comment by pidgeon777** [comment]
![image](https://github.com/user-attachments/assets/db58160c-d765-424a-8724-f0f8167b60a2)

It now works 👍.
